Formula/umple.rb

```diff
@@ -0,0 +1,23 @@
+class Umple < Formula
+  desc "Modeling tool/programming language that enables Model-Oriented Programming"
+  homepage "https://www.umple.org"
+  url "https://github.com/umple/umple/releases/download/v1.30.2/umple-1.30.2.5248.dba0a5744.jar"
+  version "1.30.2.5248.dba0a5744"
+  sha256 "11523494f9f4f360cce9226ed7f68641aa733fa13d4009f871060f9fe54c6a52"
+  license "MIT"
```

Comment:
```suggestion
  license "MIT"

```
Just one change and I think this is ready to merge. Do you mind squashing all your commits together with the first one?